marc_export/src/main/java/whelk/export/marc/ProfileExport.java

```diff
@@ -495,6 +495,7 @@ public void submit(ResultSet resultSet) throws SQLException, IOException {
         public void awaitCompletion() throws IOException {
             workQueue.awaitAll();
             out.close();
+            pool.shutdown();
```

Comment:
```suggestion
            try {
                out.close();
            } finally {
                pool.shutdown();
            }
```
(Maybe include `workQueue.awaitAll()` in the `try`, unless that failing would  make any futher recovery futile?)

Reply:
:+1: did it as a separate commit.

Reply:
Reply